ydb/_topic_writer/topic_writer_asyncio_test.py

```diff
@@ -149,6 +162,31 @@ async def test_write_a_message(self, writer_and_stream: WriterWithMockedStream):
         sent_message = await writer_and_stream.stream.from_client.get()
         assert expected_message == sent_message
 
+    async def test_update_token(self, writer_and_stream: WriterWithMockedStream):
+        assert writer_and_stream.stream.from_client.empty()
+
+        await asyncio.sleep(1)
+        expected = StreamWriteMessage.FromClient(UpdateTokenRequest(token="foo-bar"))
+        got = writer_and_stream.stream.from_client.get_nowait()
```

Comment:
you can use wait_for_fast helper:

```python
got = await wait_for_fast(writer_and_stream.stream.from_client.get())
```

It prevent wait full second during test (it will long where many test starts) and guarantee minimum count of event loops (comfort for manual debug).

Reply:
ok, will use it